tests/src/rules/first.js

```diff
@@ -11,6 +11,9 @@ ruleTester.run('first', rule, {
                   export { x, y }" })
   , test({ code: "import { x } from 'foo'; import { y } from './bar'" })
   , test({ code: "import { x } from './foo'; import { y } from 'bar'" })
+  , test({ code: "import { x } from './foo'; import { y } from 'bar'"
```

Comment:
Close! This is the same code as the previous line. the goal is to have a test that *fails* unless `'disable-absolute-first'` is passed, but passes with it.